config/config.alfajores.env

```diff
@@ -5,3 +5,6 @@ DEFAULT_LOCALE=en
 POLLING_INTERVAL=1000
 WEB3_PROVIDER_URL=https://alfajores-forno.celo-testnet.org
 DB_SECRET_NAME=projects/1067724576910/secrets/indexer-db_authentication/versions/latest
+MCUSD_ADDRESS=0x71db38719f9113a36e14f409bad4f07b58b4730b
```

Comment:
Instead of hardcoding the addresses here, we could fetch them from firebase. But since this service doesn't have connection with it, I wanted to leave it like this.
Let me know if you think it's important to change this.

Reply:
The way you did it looks good. It's better to avoid dependencies unless they are really needed 👍